pkg/config/testdata/TestLoadedConfigValidationErrors/invalid-alias.yaml

```diff
@@ -0,0 +1,28 @@
+communications: {"foo": {}}
+
+aliases:
+  kc:
+    command: kubectl
+    displayName: "Kubectl alias"
+  kgp:
+    command: kubectl get pods
+    displayName: "Kubectl Get Pods"
+  helm:
+    command: helm version
+    displayName: "Helm version"
+  eee:
+    command: ""
+
+executors:
+  'kubectl-read-only':
+    kubectl:
+      enabled: true
+  helm:
```

Comment:
Why are some keys wrapped in `'` and some aren't?

Reply:
It's mostly to distuingish configuration groups with other properties. I will unify them 👍